Add readme 1

kleberbaum opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
- Currently README doesn't serve it's purpose. #32 


**What is the new behavior (if this is a feature change)?**
- Now README serves it's purpose. #32 


**Other information**:
- Ref: https://help.github.com/en/articles/about-readmes

kleberbaum committed:
Update README.md

The README has been updated due to its lack
of explaining this repository. (It's a template README)

kleberbaum reviewed (commented):
All peta clean and tidy. ^^

Aichnerc reviewed (changes requested):
Please remove the website status, then it should be ok to merge.

kleberbaum committed:
Update README.md

The quality of the README has been improved
due to a request of @Aichnerc.
Ref: #33

Aichnerc reviewed (changes requested):
After this change, it should be okay to merge.

Aichnerc reviewed (changes requested):
After this change, it should be okay to merge.

kleberbaum committed:
Update README.md

The quality of the README has been improved again
due to a request of @Aichnerc.
Ref: #33

Aichnerc reviewed (approved):
Should be okay to merge.
